pkg/platform/abstract/platform.go

```diff
@@ -1790,6 +1818,40 @@ func (ap *Platform) enrichExplicitAckParams(ctx context.Context, functionConfig
 	return nil
 }
 
+func (ap *Platform) enrichBatchParams(ctx context.Context, functionConfig *functionconfig.Config) error {
+	for triggerName, triggerInstance := range functionConfig.Spec.Triggers {
```

Comment:
We can also just enrich it for supporting triggers instead of all of them

Reply:
@TomerShor it's more a validation than enrichment, so I'd better leave this check in one place. wdyt?